go-fips-1.22.yaml

```diff
@@ -0,0 +1,92 @@
+package:
+  name: go-fips-1.22
+  version: 1.22.0
+  epoch: 0
+  description: "the Go programming language with OpenSSL cryptography"
+  copyright:
+    - license: BSD-3-Clause
+  dependencies:
+    provides:
+      - go-fips=${{package.full-version}}
+    runtime:
+      - '!go-1.22'
+      - bash
+      - binutils-gold # Needed for cgo linking due to upstream issue #15696 which forces use of the gold linker.
+      - build-base
+      - openssl-dev # Needed for building against cryptographic packages.
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      # We always use the equivalent non-FIPS branch of Go to build this.
+      - go~1.22
+      - openssl-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://go.dev/dl/go${{package.version}}.src.tar.gz
+      expected-sha256: 4d196c3d41a0d6c1dfc64d04e3cc1f608b0c436bd87b7060ce3e23234e1f4d5c
```

Comment:
How does this packaging pull in latest golang-fips patches from https://github.com/golang-fips/go ?

Reply:
we are waiting on https://github.com/golang-fips/go/pull/160